test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetPushCommandTest.cs

```diff
@@ -773,7 +773,8 @@ public void PushCommand_PushToServerBasicAuth()
         }
 
         // Test push command to a server using basic authentication, with -DisableBuffering option
-        [SkipMono]
+        // [SkipMono]
```

Comment:
We should probably remove this.

Reply:
Sure, added details about `[SkipMono]` in issue details so next person would know about it next time.